Makefile

```
@@ -9,6 +9,9 @@ test:
 	go mod tidy -modfile=testdata/go_test.mod
 	go test ./... -modfile=testdata/go_test.mod -coverprofile=coverage.out -covermode=count
 
+benchmark:
+	go test -bench . -benchmem -benchtime 10000x
```

I would like to see go_test.mod used to reduce `rl`'s dependency packages.
This suggestion should simplify go.mod/go.sum.

```suggestion
	go mod tidy -modfile=testdata/go_test.mod
	go test -bench . -modfile=testdata/go_test.mod -benchmem -benchtime 10000x
```

---

This suggestion is not the essence of this Pull Request about improvements.
So I will fix it after the merge! Thanks!